src/main/resources/stubs/encoding/binary/binary.gobra

```diff
@@ -44,106 +44,110 @@ type ByteOrder interface {
 // type littleEndian struct{}
 // type bigEndian struct{}
 
-type LittleEndian struct {
-	// TODO: remove field
-	x int // temporary solution to side-step problems with the encoding of interfaces with fields of finite cardinality
-}
-type BigEndian struct {
-	// TODO: remove field
-	x int // temporary solution to side-step problems with the encoding of interfaces with fields of finite cardinality
-}
+const LittleEndian littleEndian = 0;
+const BigEndian bigEndian = 0;
```

Comment:
It is intentional that both have the same value?

Reply:
I did pick the same value because in the original implementation, both constants have the same value too (an instance of the empty struct).